apis/apis.json

```diff
@@ -353,7 +353,7 @@
     "id": "Google.Cloud.Firestore.V1",
     "productName": "Firestore",
     "productUrl": "https://firebase.google.com",
-    "version": "1.0.0-beta00",
+    "version": "1.0.0-beta16",
```

Comment:
beta01?

Reply:
I thought it simpler to effective "keep going" from where we'd got to with Google.Cloud.Firestore.V1Beta1. That way both Firestore packages have the same version. While that means skipping a load of betas, it's simplest if Google.Cloud.Firestore 1.0.0-betaX depends on Google.Cloud.Firestore.V1 1.0.0-betaX.